builder/osc/common/step_stop_bsu_backed_vm.go

```diff
@@ -41,7 +41,7 @@ func (s *StepStopBSUBackedVm) Run(ctx context.Context, state multistep.StateBag)
 		// does not exist.
 
 		// Work around this by retrying a few times, up to about 5 minutes.
-		err := common.Retry(10, 60, 6, func(i uint) (bool, error) {
+		err := retry.Retry(10, 60, 6, func(i uint) (bool, error) {
```

Comment:
Nit: This has a bit of stutter. What do you think renaming this method to `Run` so that it is retry.Run(...)? 

Or using the same `retry.Confg{}.Run() error {` signature that is used in Packer's retry pkg.

Reply:
done!